src/tablist.js

```diff
@@ -236,6 +236,9 @@ SideTabList.prototype = {
     if (!e.target || !SideTab.isTabEvent(e)) {
       return;
     }
+    if (e.explicitOriginalTarget.classList.contains("tab-close")) {
```

Comment:
To avoid leaking the markup implementation of SideTab in other components, I moved all class comparison operations in SideTab static methods. I think isCloseButtonEvent could be extended.